tests/virt_autotest/virt_utils.pm

```diff
@@ -349,58 +347,99 @@ sub upload_supportconfig_log {
     save_screenshot;
 }
 
+# get the guest list from the test suite settings
+sub get_guest_list {
+
+    #get the guest pattern from test suite settings
+    if (get_var('GUEST_PATTERN')) {
+        set_var('GUEST_LIST', get_var('GUEST_PATTERN'));
+    }
+    elsif (get_var('GUEST')) {
+        set_var('GUEST_LIST', get_var('GUEST'));
+    }
```

Comment:
I am lost here :)  Please explain line353-line359 ?

Reply:
Some test suites use GUEST_PATTERN while some use GUEST or GUEST_LIST in there settings to specify which guest the test check.  Thus I unify them as GUEST_LIST when figure out which guests should be downloaded and tested.  such as 
gi-guest_developing-on-host_developing-kvm            <=> GUEST_PATTERN
prj4_guest_upgrade_sles15sp2_on_developing-kvm <=> GUEST_LIST
virt-pvusb-developing-pv-on-developing-xen              <=>  GUEST

Reply:
I'll add comments.